com.Cisco.PacketTracer.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.Cisco.PacketTracer
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: packettracer
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --persist=pt # App config, logs, templates, saves stored in: ~/pt
+
+modules:
+  - name: packettracer
+    buildsystem: simple
+    build-commands:
+      - mkdir packettracer
+      - ar -x CiscoPacketTracer_821_Ubuntu_64bit.deb --output=packettracer
+      - tar -xvf packettracer/control.tar.xz --directory=packettracer
+      - tar -xvf packettracer/data.tar.xz --directory=packettracer
+      - chmod -R 755 packettracer
+      - mv packettracer/usr/share packettracer/share
+      - rm -d packettracer/usr
+      - cp -rf packettracer/. /app/
+      - sed -i 's/\/opt\/pt/\/app\/opt\/pt/g' /app/opt/pt/packettracer
+      - install -D /app/opt/pt/packettracer /app/bin/packettracer
+      - install -D /app/opt/pt/art/app.png /app/share/icons/hicolor/48x48/apps/$FLATPAK_ID.png
+      - install -D -t /app/share/metainfo $FLATPAK_ID.metainfo.xml
+      - install -D -t /app/share/applications $FLATPAK_ID.desktop
+    sources:
+      - type: file
+        url: https://gitlab.com/ezequielramos/cisco-packet-tracker/-/raw/main/CiscoPacketTracer_821_Ubuntu_64bit.deb
+        # As you can see on the official page, the sha1 being validated is the
+        # official sha1. So you can be sure that this file is the same one that
+        # cisco is providing.
+        # https://www.packettracernetwork.com/download/download-packet-tracer.html
+        sha1: d3b506411c204bac556d323d929797f33ddc1e6b
```

Comment:
sha1 are deprecated. Don't use sha1

Reply:
Their website page only lists `md5` and `sha1`. I'm currently using sha1 to prove that the downloaded file is the same one provided by their website. Should I still move to sha256?

Reply:
put a sha256.